tests/components/raycaster.test.js

```diff
@@ -7,6 +7,7 @@ suite('raycaster', function () {
     var el = this.el = document.createElement('a-entity');
     el.setAttribute('raycaster', '');
     parentEl.appendChild(el);
+    if (el.hasLoaded) { done(); }

```

Comment:
You can move the `parentEl.appendChild(el)` after the `addEventListener` instead